packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3738,15 +3738,18 @@ export class ContainerRuntime
 	}
 
 	/**
-	 * Fetches the latest snapshot from storage and uses it to refresh SummarizerNode's
-	 * internal state as it should be considered the latest summary ack.
+	 * Fetches the latest snapshot from storage to refresh the cache as a performance optimization and closes the
+	 * summarizer to reload from new state.
 	 * @param summaryLogger - logger to use when fetching snapshot from storage
-	 * @returns downloaded snapshot's reference sequence number
+	 * @returns a generic summarization error
 	 */
-	private async closeSummarizerOnSummaryStateStale(
+	private async prefetchLatestSummaryThenClose(
 		summaryLogger: ITelemetryLoggerExt,
 	): Promise<IBaseSummarizeResult> {
 		const readAndParseBlob = async <T>(id: string) => readAndParse<T>(this.storage, id);
+
+		// This is a performance optimization as the same parent is likely to be elected again, and would use its
```

Comment:
nit (can ignore): I wouldn't call it a perf optimization, it's more important than that, since it's almost certain that there's an out-of-date guy sitting in the cache which will cause us to return to this same codepath immediately. If we _don't_ cache, we have to wait for that cache to expire - so... yeah technically perf, but not in the way the comment makes me think at first glance.